lib/containers/urls.pm

```diff
@@ -51,64 +51,220 @@ sub get_opensuse_registry_prefix {
     }
 }
 
+our %registries = (
+    sle => {
+        '12-SP3' => {
+            released => sub { 'registry.suse.com/suse/sles12sp3' },
+            totest   => sub {
+                'registry.suse.de/suse/sle-12-sp3/docker/update/cr/totest/images/suse/sles12sp3';
+            },
+            available_arch => ['x86_64', 'ppc64le', 's390x']
+        },
+        '12-SP4' => {
+            released => sub { 'registry.suse.com/suse/sles12sp4' },
+            totest   => sub {
+                'registry.suse.de/suse/sle-12-sp4/docker/update/cr/totest/images/suse/sles12sp4';
+            },
+            available_arch => ['x86_64', 'ppc64le', 's390x']
+        },
+        '12-SP5' => {
+            released => sub { 'registry.suse.com/suse/sles12sp5' },
+            totest   => sub {
+                'registry.suse.de/suse/sle-12-sp5/docker/update/cr/totest/images/suse/sles12sp5';
+            },
+            available_arch => ['x86_64', 'aarch64', 'ppc64le', 's390x']
+        },
+        '15' => {
+            released => sub { 'registry.suse.com/suse/sle15:15.0' },
+            totest   => sub {
+                'registry.suse.de/suse/sle-15/update/cr/totest/images/suse/sle15:15.0';
+            },
+            available_arch => ['x86_64', 'ppc64le', 's390x']
+        },
+        '15-SP1' => {
+            released => sub { 'registry.suse.com/suse/sle15:15.1' },
+            totest   => sub {
+                'registry.suse.de/suse/sle-15-sp1/update/cr/totest/images/suse/sle15:15.1';
+            },
+            available_arch => ['x86_64', 'aarch64', 'ppc64le', 's390x']
+        },
+        '15-SP2' => {
+            released => sub { 'registry.suse.com/suse/sle15:15.2' },
+            totest   => sub {
+                'registry.suse.de/suse/sle-15-sp2/update/cr/totest/images/suse/sle15:15.2';
+            },
+            available_arch => ['x86_64', 'aarch64', 'ppc64le', 's390x']
+        },
+        '15-SP3' => {
+            released => sub { 'registry.suse.com/suse/sle15:15.3' },
+            totest   => sub {
+                'registry.suse.de/suse/sle-15-sp3/update/cr/totest/images/suse/sle15:15.3';
+            },
+            available_arch => ['x86_64', 'aarch64', 'ppc64le', 's390x']
+        },
+        '15-SP4' => {
+            released => sub { '' },
+            totest   => sub {
+                'registry.suse.de/suse/sle-15-sp4/update/cr/totest/images/suse/sle15:15.4 ';
```

Comment:
i have replaced it. I might be wrong here but i think sle15sp4 should test `suse/sle-15-sp4/update/cr/totest/images/suse/sle15`. 
registry.s.d doesnt provide `suse/sle-15-sp4/ga/test/images/suse/sle15` (unless i have really overseen it) anyway.

Reply:
I trust you!

Reply:
Yes, Marcus Meissner enabled this project for 15-SP4 images. I am not sure if it's wrong or not, cause we were told to use  `registry.suse.de/suse/sle-15-sp4/ga/test/images/suse/sle15:15.4` in the beginning... Let's keep it like this and change it later on if we are told so.

Reply:
From 15 SP3 on, the locations before and after GA (maintenance testing) are different. For now it has to be `registry.suse.de/suse/sle-15-sp4/ga/test/images/suse/sle15` for SP4.

Reply:
Why is this resolved? Let's just update it to the right location.

Reply:
@Vogtinator [from another pr which uses that registry](https://openqa.suse.de/tests/7061832#step/buildah_podman/83) you can see that it fails there. but i found that the path is also wrong. should it be `suse/sle-15-sp4/ga/images/suse/sle15` instead of `registry.suse.de/suse/sle-15-sp4/ga/test/images/suse/sle15`?

Reply:
No, it needs to be `registry.suse.de/suse/sle-15-sp4/ga/test/images/suse/sle15`. It was only fixed in IBS yesterday and there's no build released to test yet.

Reply:
So, let's then update it to `registry.suse.de/suse/sle-15-sp4/ga/test/images/suse/sle15:15.4`. The next IBS build for 15-SP4 should contain the image already.

Reply:
**this change is ready for review**